openstack/manila/deployment.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: manila-provisioner
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: manila-provisioner
+    spec:
+      containers:
+      - name: manila-provisioner
+        image: "quay.io/external_storage/manila-provisioner:latest"
+        imagePullPolicy: IfNotPresent
+        env:
+        - name: OS_AUTH_URL
+          value: http://192.168.2.16:5000/v2.0
+        - name: OS_USERNAME
+          value: admin
+        - name: OS_PASSWORD
```

Comment:
Please consider authenticating using a trustid, it is supported by gophercloud.

Reply:
@strigazi , glad to see you here, from OpenStack magnum to Kubernetes!

For the problem, absolutely I should use trustid. learn from magnum, it should be able to mount cloud config file which to manila provisioner. what it looks like now is what copied from standalone-cinder. I will fix it in next patch